CMakeLists.txt

```diff
@@ -5,22 +5,16 @@ enable_testing()
 
 set(CMAKE_OSX_DEPLOYMENT_TARGET "10.12" CACHE STRING "Minimum OS X deployment version")
 
-project(VULKAN_PROFILES LANGUAGES CXX C)
+project(VULKAN_PROFILES LANGUAGES CXX)
```

Comment:
It's best practice to minimize the compilers used for your project. It complicates the CMake code.